WIP-CHANGELOG.md

```diff
@@ -65,6 +65,7 @@
 - increased the influxDB pvc size [#739](https://github.com/elastisys/compliantkubernetes-apps/pull/739)
 - Exposed velero's backup timetolive for both sc and wc.
 - disabled internal database for InfluxDB
+- kubeconfig.bash have been edited to work with the new 'secret' ck8s-ops structure.
```

Comment:
I guess you mean
```suggestion
- kubeconfig.yaml have been edited to work with the new 'secret' ck8s-ops structure.
```

Reply:
This is actually modifying the `bin/kubeconfig.bash` script.

Reply:
Riiiight! You're very much correct